modules/03 Front End Development/Assignments/02/answer.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Images | Favorites</title>
+</head>
+<body>
+    <a href="https://en.wikipedia.org/wiki/San_Francisco"> 
+        <img width="400px" height="400px" src="https://www.usnews.com/dims4/USNEWS/d2c027c/2147483647/resize/1200x%3E/quality/85/?url=http%3A%2F%2Fmedia.beam.usnews.com%2F17%2F5c%2Fe08bd2a347be848e3dd4e98d88d9%2F2-30.%20San%20Francisco-Getty.jpg">
```

Comment:
You defined "width" and "height" incorrectly. Try not to run too fast and use only the things we learn in the class